ci-operator/config/openshift/openshift-tests-private/AGENTS.md

```diff
@@ -0,0 +1,251 @@
+
+This file provides guidance for AI agents (like Claude Code) when working with openshift-tests-private CI configurations. It complements the human-focused README.md with agent-specific instructions and patterns.
+
+## File Location
+This file is located at: `ci-operator/config/openshift/openshift-tests-private/AGENTS.md`
+Prow CI configuration files are located in `ci-operator/config/openshift/openshift-tests-private`
+
+## Folder Structure Overview
+```
+openshift-tests-private/
+├── tools/                          # Contains additional scripts for maintaining configuration files
+│   ├── OWNERS                      # The tools folder's reviewers and approvers for github commit
+│   ├── update-cron-entries.py      # Use this script to update all the job cron settings
+│   ├── add_CPOU_upgrade_jobs.py
+│   ├── add_missed_upgrade_jobs.py
+│   ├── generate-cron-entry.sh
+│   ├── get_missed_upgrade_jobs.py
+│   └── jobs_heatmap_view.py
+├── README.md                       # Prow CI documentation
+├── OWNERS                          # The openshift-tests-private folder's reviewers and approvers for github commit
+├── openshift-openshift-tests-private-release-<TARGET_VERSION>__<CPU_ARCH>-<IMAGE_STREAM>-<TARGET_VERSION>-upgrade-from-<IMAGE_STREAM>-<INITIAL_VERSION>.yaml
+├── openshift-openshift-tests-private-release-<TARGET_VERSION>__<CPU_ARCH>-<IMAGE_STREAM>-<TARGET_VERSION>-cpou-upgrade-from-<INITIAL_VERSION>.yaml
+└── openshift-openshift-tests-private-release-<VERSION>__<CPU_ARCH>-rollback-<IMAGE_STREAM>.yaml
+```
+
+## Glossary
+
+- **Image Stream**: The OpenShift release channel (e.g., `nightly`, `stable`, `candidate`)
+- **Initial Version**: The OpenShift version before the upgrade
+- **Target Version**: The OpenShift version after the upgrade
+- **CPU Architecture**: `amd64`, `arm64`, or `multi` (multi-arch)
+- **f28/f60**: Job frequency in days (`f` stands for `frequency`, f28 = every 28 days, f60 = every 60 days)
+- **Profile**: A combination of features/configurations for testing (e.g., ovn-ipsec, fips, proxy)
+- **EOL**: End of Life - version no longer supported by Red Hat
+- **SNO**: Single Node OpenShift
+- **STS**: AWS Security Token Service
+- **UPI**: User Provisioned Infrastructure
+- **CPOU**: Control Plane Only Updates
+- **TP**: TechPreview features
+
+## Upgrade File Conventions
+
+### 1. Upgrade File Definition
+- Y stream upgrade files, Z stream upgrade files and Chain upgrade files' name contain `upgrade`
+- CPOU upgrade file's name contains `cpou` and `upgrade`
+- Rollback upgrade file's name contains `rollback`
+
+### 2. Target Version
+- The first version behind `openshift-openshift-tests-private-release-` in the upgrade file name is the target version
+- All jobs in an upgrade file are used to test the target OpenShift version
+
+### 3. CPOU Upgrade
+- CPOU is an abbreviation for Control Plane Only Updates.
+- CPOU is a kind of special upgrade, its file name contains both `cpou` and `upgrade`.
```

Comment:
shall we remove this line as it's already added in https://github.com/openshift/release/pull/71613/files#diff-56f2a8ee72ea7aed1ef4cee3a3d0b228d8e1c9518c86ce946cb60ad236173caaR46?

Reply:
sure